applications/RomApplication/tests/NonlinearTestFiles/MainKratosROM.py

```diff
@@ -0,0 +1,184 @@
+from __future__ import print_function, absolute_import, division #makes KratosMultiphysics backward compatible with python 2.6 and 2.7
+
+import KratosMultiphysics
+import KratosMultiphysics.ConvectionDiffusionApplication
+import KratosMultiphysics.RomApplication as romapp
+
+from KratosMultiphysics.assign_scalar_variable_process import AssignScalarVariableProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.apply_thermal_face_process import ApplyThermalFaceProcess
+from KratosMultiphysics.ConvectionDiffusionApplication.convection_diffusion_analysis import ConvectionDiffusionAnalysis
+
+import h5py
+import json
+import sys
+import time
+
+class ConvectionDiffusionAnalysisWithFlush(ConvectionDiffusionAnalysis):

```

Comment:
As far as I understand, this is the stage class you need to perform the ROM. However, there are a lot of things in here that are hardcoded for the thermal problem. Although so far it is only used in the tests, we need to find a solution for the interface.

Reply:
Aside of this, I think this should go to the `python_scripts` folder and that the class must be renamed (now it matches the one in the convection diffusion application and it has no flush).

Reply:
The class was simplified. Indeed, similar files are to be added to the python_scripts folder in following PRs